quinn-proto/src/connection/mod.rs

```diff
@@ -2820,12 +2822,17 @@ where
         self.streams.alloc_remote_stream(&self.peer_params, dir);
     }
 
-    fn add_read_credits(&mut self, id: StreamId, transmit_max_stream_data: bool) {
+    fn add_read_credits(
+        &mut self,
+        id: StreamId,
+        transmit_max_stream_data: frame::WantTransmit,
```

Comment:
Let's just import `WantTransmit` from `frame` directly.

Also, I feel like `ShouldTransmit` better matches the semantics than `WantTransmit`, do others feel the same?

Reply:
I'm weakly in favor.